cli/locales/de/messages.gotext.json

```diff
@@ -15917,64 +15917,96 @@
       "fuzzy": true
     },
     {
-      "id": "/var/log/syslog could not be opened or does not exist: {Err}",
-      "message": "/var/log/syslog could not be opened or does not exist: {Err}",
-      "translation": "'/var/log/syslog' konnte nicht geöffnet werden oder ist nicht vorhanden, {Err}",
+      "id": "{SysLogPath} could not be opened or does not exist: {Err}",
```

Comment:
These files are generated by the go text tools, please do not change any files that end in gotext*.json. The files with non-English translations are also tool generated, IBM donates them to this project. Please remove them all from this PR.

Reply:
Removed all changes to gotext files